src/Practitioner/ReviewPatients/checkWasToday.js

```diff
@@ -0,0 +1,7 @@
+import { DateTime } from 'luxon';
+
```

Comment:
I would move this function here

src/Utility/TimeUtils.js

Trying to collect common util code there

Reply:
Oh sorry this showed up in the wrong spot. I meant this whole file

Reply:
Moved, now imported from TimeUtils, and deleted checkWasToday